app/bundles/InstallBundle/Controller/InstallController.php

```diff
@@ -99,37 +102,43 @@ public function stepAction($index = 0)
                         $dbParams = (array) $formData;
 
                         $messages = $this->installer->createDatabaseStep($step, $dbParams);
-                        if (is_bool($messages) && true === $messages) {
-                            // Refresh to install schema with new connection information in the container
-                            return $this->redirect($this->generateUrl('mautic_installer_step', ['index' => 1.1]));
-                        } elseif (is_array($messages) && !empty($messages)) {
+                        if (!empty($messages)) {
                             $this->handleInstallerErrors($form, $messages);
+                            break;
                         }
-                        break;
 
+                        // XXX Regression: we used to also get this if database created but configuration not saved
+                        $schemaHelper             = new SchemaHelper($dbParams);
+                        $formData->server_version = $schemaHelper->getServerVersion();
```

Comment:
This was removed in https://github.com/mautic/mautic/pull/9739 so we should not be re-adding it here

Reply:
I removed these lines, I do not think we need them given the PR #9739 I tested locally on both CLI and web and it worked as expected, I removed them